framework/db/oci/Command.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+
+namespace yii\db\oci;
+
+
+class Command extends \yii\db\Command
+{
+    /**
+     * {@inheritdoc}
+     */
+    protected function bindPendingParams()
+    {
+        foreach ($this->_pendingParams as $name => $value) {
+            if($value[1] === \PDO::PARAM_STR) {
+                $clonedValue = clone $value[0];
```

Comment:
It looks suspicious. `clone` works with objects only so the statement would fail if I'll do

```php
$command->bindValues(['myParam' => 'myValue']);
```